server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -691,6 +691,14 @@ public static IndexMergePolicy fromString(String text) {
         Property.Dynamic
     );
 
+    public static final Setting<Integer> INDEX_CONCURRENT_SEGMENT_SEARCH_MAX_SLICE_COUNT = Setting.intSetting(
+        "index.search.concurrent.max_slice_count",
+        0,
+        0,
+        Property.Dynamic,
```

Comment:
also, what would 0 as the default or minimum slice count mean? shouldn't it be 1?

Reply:
0 means Lucene default slice count

Reply:
i see, basically it would be helpful if user wants to use lucene default for some of the indices. lets add a comment as well for this then.

Reply:
lets use these: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/search/SearchService.java#L264-L265